spec/fortnox/api/mappers/base/from_json_spec.rb

```diff
@@ -24,10 +24,10 @@
       {
         'Product' => {
           '@url': 'someurl@example.com',
-          'Name': 'Ford Mustang',
-          'VAT': 30_000,
-          'Categories': [{ 'Name' => 'Cars', 'ID' => '1' }, { 'Name' => 'Fast cars', 'ID' => '2' }],
-          'Designer': { 'Name' => 'John Najjar', 'ID' => '23' }
+          Name: 'Ford Mustang',
+          VAT: 30_000,
+          Categories: [{ 'Name' => 'Cars', 'ID' => '1' }, { 'Name' => 'Fast cars', 'ID' => '2' }],
```

Comment:
I do though strongly question why it hasn't applied the same logic to these???
```
Categories: [{  Name: 'Cars', ID: '1' }, { Name: 'Fast cars', ID: '2' }],
```
That makes a lot more sense to me, in context...

Reply:
I'll fix it. I guess it just didn't catch this...